src/region/RegionList.tsx

```diff
@@ -34,14 +36,21 @@ export function sortRegion({ target: targetA }: AnnotationRegion, { target: targ
 export function RegionList({ activeId, annotations, className, onSelect = noop }: Props): JSX.Element {
     const [isListening, setIsListening] = React.useState(true);
     const rootElRef = React.createRef<HTMLDivElement>();
+    const rotation = Redux.useSelector(getRotation);
 
     // Document-level event handlers for focus and pointer control
-    useOutsideEvent('click', rootElRef, (): void => onSelect(null));
-    useOutsideEvent('mousedown', rootElRef, (): void => setIsListening(false));
+    useOutsideEvent('mousedown', rootElRef, (): void => {
+        onSelect(null);
+        setIsListening(false);
```

Comment:
Interestingly, this was causing a re-render and dropping the `click` handler above when only one RegionList was being rendered.